docs/students/signing-up.md

```diff
@@ -1,5 +1,22 @@
 # Signing Up 
 
-How to sign up for Bible First Online? You need a signup key. Then you can sign up. No key, no enchilada.
+If a Bible First coach has invited you to sign up, then hopefully you've been given either a web address or a signup key. If you've been given a web address, just visit it and follow the instructions.
```

Comment:
The term "web address" threw me for a minute when I first saw it. Made me think of a web address for a site that's mine, which didn't make sense. (Think: here's the web address for your new Plexus back office or something.) Maybe it would be clearer if we called it a "signup link"?

Reply:
Changed it to "signup link" or just "link", since the word "signup" was being used a lot.